index.php

```diff
@@ -75,7 +76,15 @@
   		</div>
```

Comment:
please make this code look like the format which will make others to understand easily